docs/webapp/applications/apps_aws_autoscaler.md

```diff
@@ -171,6 +172,15 @@ auto_scaler.v1.aws {
 }
 ```
 
+#### Configure Instances Spawned by the Autoscaler
+To configure instances spawned by the autoscaler, do any of the following:
+* Add the configuration in the `extra_clearml_conf` field of the configuration vault
+* Run the Autoscaler using a [ClearML Service Account](../../webapp/webapp_profile.md#service-accounts). Add the 
+configuration to the service account's configuration vault, and set the autoscaler to run under that account
+in the `Run with Service Account` field
+* Admins can add the configuration to a [ClearML Administrator Vault](../../webapp/webapp_profile.md#administrator-vaults)
+and link the vault with the [user group](../../webapp/webapp_profile.md#user-groups) that includes the user running the 
```

Comment:
@pollfly "with the user group" -> "with a user group" (users can be in multiple groups)